scripts/add_title_geo_and_CNR_to_first_line.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+import pandas as pd
+
+from scripts.task import Task
+
+
+class AddTitleGeoCNRToFirstLine(Task):
+
+    def __init__(self, input_dir: str):
+        super().__init__()
+        self.input_dir = input_dir
+
+    def _execute(self):
+        assert isinstance(self.shared_resource, pd.DataFrame)
+        for filename in os.listdir(self.input_dir):
+            if filename.endswith(".txt"):
+                judgement = os.path.join(self.input_dir, filename)
+                case_number = filename.split('_')[0]
+                case = self.shared_resource[self.shared_resource['case_no'] == case_number]
+                title = self._get_case_title(case)
+                with open(judgement, "r+") as input_file:
+                    judgement = input_file.read()
+                    input_file.seek(0)
+                    input_file.write("{} ||| {}".format(title, judgement))
```

Comment:
are we adding the title in the name of the file or the content itself?

Reply:
To the content. First few characters of the content are shown as title.